src/main/resources/templates/tutorials.html

```diff
@@ -0,0 +1,146 @@
+                    </div>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                   <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                    <div class="col-sm-6">
+                        <img preload="true" minimal-lightbox="images/register2.png" src="images/register2.png" width="500" height="315"/></div>
+                    <div class="col-sm-6">
+                        <div class="heading col-sm-12 col-sm-offset-1">
+                            <p>&nbsp;</p>
+                            <p> 1) Fill in all required information here. ALL fields are required. </p>
+
+                        </div>
+                    </div>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+
+                    <div class="col-sm-6">
+                        <img preload="true" minimal-lightbox="images/register3.png" src="images/register3.png" width="500" height="315"/></div>
+                    <div class="col-sm-6">
+                        <div class="heading col-sm-12 col-sm-offset-1">
+                            <p>&nbsp;</p>
+                            <p> 1) Choose to create a new team, or to join an existing team. </p>
+
+                        </div>
+                    </div>
+                </div>
+
+                <h2>3) How to Join a Team </h2>
+                <div class="heading col-sm-12 ">
+                    <div class="col-sm-6">
+                    <img preload="true" minimal-lightbox="images/jointeam.png" src="images/jointeam.png" width="500" height="315"/></div>
+                    <div class="col-sm-6">
+                        <div class="heading col-sm-12 col-sm-offset-1">
+                            <p>&nbsp;</p>
+                            <p> 1) At the “Teams” page you can view information about your team as well as pending requests for participation.</p>
+                            <p> 2) To view specific team information, click on the team name. </p>
+                            <p> 3) You can also create a new team or join a team here.</p>
+                        </div>
+                    </div>
+                    <p>&nbsp;</p>
+                    <p>&nbsp;</p>
+                </div>
+
+
+            <div class="cc-wraper">
+                <div class="cc-contactpop fixed">
+                    <h1>Do you need Help? <span>Get in touch.</span></h1>
+                    <div class="renseignement">
+                        <i class="fa fa-times"></i>
+                        <p>We will get back to you as soon as possible. </p>
+                    </div>
+                    <form>
+                        <ul>
+                            <li><i class=""></i> <span> Phone: </span>+65 6601 1056</li>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)